package.json

```diff
@@ -32,6 +33,10 @@
     "eslint-plugin-import": "^2.25.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.1",
-    "jest": "^27.3.1"
+    "jest": "^27.3.1",
+    "minimist": "^1.2.6"
+  },
+  "resolutions": {
+    "minimist": "^1.2.6"
```

Comment:
Refer to following article.

https://zenn.dev/hellorusk/articles/54b964379541a0